modules/power_z_unsupported_known_issues.adoc

```diff
@@ -0,0 +1,18 @@
+[id="power z known issues_{context}"]
+== IBM Z and LinuxONE known issues. 
+
+  * Only supported on {product-title} 4.6 
+
+== IBM Power Systems known issues.
+  
+  * Only supported on {product-title} 4.6
+
+== The following features are unsupported on IBM Z:
+
+** Jaeger streaming via AMQ/Kafka
+** WebAssembly
```

Comment:
I would move the non support of WebAssembly into the Service Mesh known issues module (ossm-rn-known-issues.adoc) in the top section before the heading.